support-frontend/knip.config.js

```diff
@@ -8,6 +8,26 @@ module.exports = {
 	entry: flattenedEntryPoints,
 	project: ['**/*.{js,jsx,ts,tsx,scss}!'],
 	ignoreExportsUsedInFile: true,
+	ignoreDependencies: [
+		// used in package.json
+		'@guardian/browserslist-config',
+		'lint-staged',
+		// used by scripts in package.json
+		'webpack-cli',
+		'webpack-dev-server',
+		// could be used to run Chromatic locally
```

Comment:
is this accurate? I tried to remove it and the github actions still work successfully so I'm not sure it's strictly needed in package.json, but seems wrong to remove